### ISSUE TITLE
[Bug]: Sigil has some severe contrast ratio issues

### ISSUE THREAD

**Issue by Lululalu** [Sigil-Ebook/Sigil#849]
### Bug Description

<img width="475" height="89" alt="Image" src="https://github.com/user-attachments/assets/d04e7963-7770-42e9-8366-96dae8067620" />

<img width="292" height="174" alt="Image" src="https://github.com/user-attachments/assets/8612ad6b-1e31-45e4-9333-bcde65ee973e" />

Just check the screenshot above. It's barely usable.

### Platform (OS)

macOS

### OS Version / Specifics

15.7.4

### What version of Sigil are you using?

2.7.0

### Any backtraces or crash reports

```shell

```

**Comment by kevinhendricks** [comment]
You control all CodeView syntax and text colours in Sigil preferences.
Closing this as not a bug.

**Comment by Lululalu** [comment]
What an arrogant attitude. The default is bad and should be fixed. People use Sigil to work, not to waste time fiddling with colors, fixing an interface that, by default has serious accessibility issues.

**Comment by kevinhendricks** [comment]
This is what it looks like on my Mac in dark mode.  We can not pick defaults that work across all platforms and themes but they work fine on my Mac Desktop. Changing the highlight colour is trivial and remembered forever.

<img width="622" height="238" alt="Image" src="https://github.com/user-attachments/assets/8dc372e9-dcca-434c-b9cf-2578426c5000" />

**Comment by kevinhendricks** [comment]
What you claim is not what I am seeing on my Mac at all.  So actually try reading the Sigil's User guide to and learn how to use Sigil Preferences to make it any way you want.

As as a long time unpaid volunteer developer for Sigil (over a decade), I think I my attitude is just fine and so was my response.  
No more personal attacks will be tolerated.
